### PR TITLE
[repo] Remove Flutter code from the workspace

### PR DESCRIPTION
**Pull request by sjchmiela**
# Why

Even though it was a really interesting project (and big thanks to @FiberJW for taking it on and delivering! 💯) we no longer plan to maintain Unimodules integration for platforms other than React Native due to reduced bandwidth and increased focus.

# How

- removed flutter packages from the repository
- searched for `flutter` keyword throughout the repository, removed applicable hits (left: `Pods/SDWebImage` and `docs/react-native`
- ran `find . | grep flutter` and `…pubspec` in the root of the repository to ensure no leftovers are… left over.

# Test Plan

None.

### CODE REVIEW

**Review by ide — APPROVED** [review]

File: packages/expo-ads-admob/android/build.gradle
Comment (EvanBacon) [inline packages/expo-ads-admob/android/build.gradle:55]:
```suggestion
      "'unimodules-core.gradle' was not found in the usual React Android dependency location. " +
```

🤔

**Review by tsapeta — APPROVED** [review]

**Review by bbarthec — APPROVED** [review]

File: packages/expo-ads-admob/android/build.gradle
Comment (sjchmiela) [reply packages/expo-ads-admob/android/build.gradle:55]: It's a tough one to reason about, but as I can see it, `unimodules-core` is like a _React Native dependency_, not _React Android dependency_ and we are looking for it in a place usual for a _React Native project_, not a _React Android project_. In general I think _React Android_ is a term more commonly known among RN contributors than developers.

Comment (sjchmiela) [reply packages/expo-ads-admob/android/build.gradle:55]: I'll let myself merge this PR, but if you feel strongly about this suggestion let's come back to this discussion online!

**Pull request merged**
